### ISSUE TITLE
creating arbitrary classnames from js objects

### ISSUE THREAD

**Issue by XEngine** [Izaanaami/tailwind-parser-js#1]
Hey! Awesome library!

I'm having very hard time to figuring your parser function in order to open a PR so I decided to open an issue about it. 
Parsing classnames like "gap-[20px]" into JS Object is perfectly working but turning the same JS Object into classname is not working.
I am guessing arbitrary coversation not implemented yet.

I am terribly sorry not opening a PR

**Comment by Izaanaami** [comment]
yes I have not implemented arbitrary className creation out of javascript objects yet. I will add that feature as soon as I have free time on my hand. also you are more than welcome to open an pull request and implement it yourself, as it doesn't require much work and most of it is already done.

**Comment by XEngine** [comment]
https://github.com/Izaanaami/tailwind-parser-js/pull/3
